src/main/java/rocks/inspectit/gepard/agent/resolver/ConfigurationResolver.java

```diff
@@ -77,4 +81,14 @@ private InstrumentationConfiguration getConfiguration() {
   private boolean shouldIgnore(String fullyQualifiedName) {
     return fullyQualifiedName.contains("$$Lambda") || fullyQualifiedName.startsWith("[");
   }
+
+  public ElementMatcher.Junction<MethodDescription> getElementMatcherForType(TypeDescription type) {
+    InstrumentationConfiguration configuration = getConfiguration();
+    Scope scope = configuration.getScopeByFqn(type.getName());
+    String methodName = scope.getMethod();
+    if (methodName == null) {
```

Comment:
Can you change it to `Objects.isNull(methodName)` since it's used throughout the project now. It looks more clean.